prac_05/state_names.py

```diff
@@ -4,15 +4,19 @@
 File needs reformatting
 """
 
-# TODO: Reformat this file so the dictionary code follows PEP 8 convention
-CODE_TO_NAME = {"QLD": "Queensland", "NSW": "New South Wales", "NT": "Northern Territory", "WA": "Western Australia",
-                "ACT": "Australian Capital Territory", "VIC": "Victoria", "TAS": "Tasmania"}
+CODE_TO_NAME = {"QLD": "Queensland", "NSW": "New South Wales", "NT": "Northern Territory",
+                "WA": "Western Australia", "ACT": "Australian Capital Territory",
+                "VIC": "Victoria", "TAS": "Tasmania"}
 print(CODE_TO_NAME)
 
+# Print all states.
+for state in CODE_TO_NAME:
+    print("{:3} is {}".format(state, CODE_TO_NAME[state]))
+
 state_code = input("Enter short state: ")
```

Comment:
Maybe use upper to change the users input as soon as its entered.

Reply:
Good point, forgot you could add the upper method to input.

Reply:
Good point, forgot you could use upper method on input.